services/tasks-svc/repos/patient_repo.sql

```diff
@@ -0,0 +1,18 @@
+-- name: CreatePatient :exec
+INSERT INTO patients
+	(id, human_readable_identifier, notes, created_at, updated_at)
+VALUES ($1, $2, $3, $4, $5);
+
+-- name: UpdatePatient :exec
+UPDATE patients
+SET human_readable_identifier = coalesce(sqlc.narg('human_readable_identfier'), human_readable_identifier),
+    notes = coalesce(sqlc.narg('notes'), notes),
+	updated_at = coalesce(sqlc.narg('updated_at'), updated_at),
```

Comment:
formatting

Reply:
Seems to be a GitHub displaying error, is not indented in the file itself

Reply:
check the whitespace of the notes line